bcda/suppression/suppression.go

```diff
@@ -133,13 +135,21 @@ func (importer OptOutImporter) ImportSuppressionData(metadata *optout.OptOutFile
 			importer.Logger.Error(err)
 			return err
 		}
+		switch suppression.PrefIndicator {
+		case "Y":
+			optInCount++
+		case "N":
+			optOutCount++
+			return nil
```

Comment:
do we need the return here? seems like it'll fall down into the one right below this.

Reply:
thanks for catching that! not sure why that's in there..